urlResolvers/github.js

```diff
@@ -0,0 +1,42 @@
+const URLResolver = require('./urlResolver');
+const GitHubAPIHandler = require('../sourceHandlers/github');
+
+const stripHttp = /^(http(s)?:\/\/)?(www.)?/;
+const urlRegex = /^github\.com\/([\w,-]+)\/([\w,-]+)/;
+
+const apiBaseUrl = 'https://api.github.com/';
```

Comment:
Really really long shot here, but if someone was hosting their own instance of github enterprise we could reuse the handler and just pass in a different base url. More info [here](https://developer.github.com/v3/enterprise/).